packages/legend-extension-external-store-service/src/components/ESService_LegendStudioPlugin.tsx

```diff
@@ -79,6 +81,26 @@ export class ESService_LegendStudioPlugin
     ];
   }
 
+  getExtraElementIconGetters(): ElementIconGetter[] {
+    return [
+      (type: string): React.ReactNode | undefined => {
+        if (type === SERVICE_STORE_ELEMENT_TYPE) {
+          return (
+            <div
+              className="icon"
+              style={{
+                color: 'var(--color-light-grey-50)',
```

Comment:
could you take screenshot of what this look like in the explorer and dropdown menu?

Reply:
![pic1](https://user-images.githubusercontent.com/87973126/145417574-bee9a7e5-ccdb-4057-81bd-b6579ba6f060.PNG)
![pic2](https://user-images.githubusercontent.com/87973126/145417602-6a52a821-4729-46f8-8c95-383044b75dcc.PNG)